apps/vc-api/test/vc-api/workflows/resident-card/resident-card.e2e-suite.ts

```diff
@@ -94,9 +95,16 @@ export const residentCardWorkflowSuite = () => {
       didAuthStepId
     );
 
-    // As the issuer, check the result of the transaction verification
-    expect(stepSubmission.presentationSubmission.verificationResult.verified).toBeTruthy();
-    expect(stepSubmission.presentationSubmission.verificationResult.errors).toHaveLength(0);
+    // As the issuer, check the result of the step verification
+
+    // TODO: revisit the isComplete test
+    // expect(stepSubmission.step.isComplete).toBeTruthy();
+
+    if (stepSubmission.step.type === 'QueryExchangeStep') {
+      const queryExchangeStep = stepSubmission.step as QueryExchangeStep;
+      expect(queryExchangeStep.presentationSubmission.verificationResult.errors).toHaveLength(0);
+    }
+    // expect(stepSubmission.stepResponse.verificationResult.verified).toBeTruthy();
```

Comment:
I'm wondering if we can uncomment this one too now.

Reply:
we can uncomment this one, and it works